packages/zosfiles/src/methods/utilities/Utilities.ts

```diff
@@ -78,8 +86,12 @@ export class Utilities {
         if (codeset) {
             payload.codeset = codeset;
         }
-        await Utilities.putUSSPayload(session, ussFileName, payload);
 
+        if (responseTimeout != null && responseTimeout >= 5) {
+            await Utilities.putUSSPayload(session, ussFileName, payload, responseTimeout);
+        } else {
+            await Utilities.putUSSPayload(session, ussFileName, payload)
+        }
```

Comment:
I don't think there would be any harm in just calling the function since you handled the responseTimeout in the putUSSPlayload method 🙏 

```suggestion
        await Utilities.putUSSPayload(session, ussFileName, payload, responseTimeout);
```